tests/docker/.env.sh

```diff
@@ -1,15 +1,17 @@
 #!/usr/bin/env bash
 
-export PY_DOCKER_SOURCE="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+export PY_DOCKER_SOURCE=$(pwd)
```

Comment:
@emasab would be better to comment on this and next file.

Reply:
hmm , should I ask it in his office hours ?